### ISSUE TITLE
[Azure Toolkit][A11yMAS][Accessibility][ColourContrast][MAS1.4.3][IntelliJ][Wipro]: Colour contrast ratio is below 4.5:1 for selected created "Key" and "Value" in show properties window.

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/azure-tools-for-java#1423]
**Repro Steps:**

1. In "Azure Explorer" , select the created app under web apps and use shift+F10 to open the context menu.
2. Navigate to and select "Show properties" menu item .
3. Properties window will open.
4. Tab to "key" and "Value" and  observe the colour contrast ratio.

**Expected Result:**
Colour contrast ratio should be greater than or equal to 4.5:1 for  selected created "Key" and "Value" in show properties window.

**Actual Result:**
 Colour contrast ratio is  4.4:1 for selected created "Key" and "Value"  in show properties window.

**Platform:**
IntelliJ IDEA 2017.3.2 (Community edition)
Build# IC-173.4127.27
JRE:1.8.0_152-release-1024-b8amd64
JVM: OpenJDK 64 Bit Server VM by Jetbrains.
**OSBuild Version:** Version 1709(Os Build 16299.192)
**CCA version:** 2.5.0.0

![cca in show properties window](https://user-images.githubusercontent.com/22956130/35718484-a22d398a-0799-11e8-9b67-de9664cbf01e.png)

**Comment by Eskibear** [comment]
It has external dependency, IntelliJ IDEA handles its own theme, refer to:
https://youtrack.jetbrains.com/issue/IDEA-132299

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749865
